mmv1/products/dns/terraform.yaml

```diff
@@ -16,6 +16,15 @@ overrides: !ruby/object:Overrides::ResourceOverrides
   ManagedZone: !ruby/object:Overrides::Terraform::ResourceOverride
     id_format: 'projects/{{project}}/managedZones/{{name}}'
     examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "dns_managed_zone_quickstart"
+        primary_resource_id: "default"
+        skip_docs: true
+        vars:
+          dns_compute_instance: "dns-compute-instance"
+          allow_http_traffic: "allow-http-traffic"
+          example_zone_googlecloudexample: "example-zone-googlecloudexample"
+        min_version: beta
```

Comment:
test is failing because terraform is trying to import the managed zone and compare the `force_destroy` field (which doesn't exist in the API). We'll need to ignore the field with:
```
        ignore_read_extra:
          - "force_destroy"
```